### PR TITLE
Add missing copyright information

### PR DESCRIPTION
**Pull request by TAbhay**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

**Test plan (required)**

**Closing issues**

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Cutter
+Upstream-Contact: RizinOrg <info@rizin.re>
+Source: https://github.com/rizinorg/cutter
+
+# Sample paragraph, commented out:
```
Comment (XVilka) [inline .reuse/dep5:6]: Just remove the sample lines.

File: src/Main.cpp [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+//
```
Comment (XVilka) [inline src/Main.cpp:2]: No need for space between them.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Comment by XVilka** [comment]
Now you need to add these lines to the rest of the files. But before that, lets wait what @thestr4ng3r @xarkes @karliss @ITAYC0HEN say on the copyright contents/format.

**Comment by TAbhay** [comment]
> Now you need to add these lines to the rest of the files. But before that, lets wait what @thestr4ng3r @xarkes @karliss @ITAYC0HEN say on the copyright contents/format.

Okay

**Comment by karliss** [comment]
Probably needs additional entries for the fonts and icons.

File: LICENSES/GPL-3.0-or-later.txt [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+Finally, every program is threatened constantly by software patents. States should not allow patents to restrict development and use of software on general-purpose computers, but in those that do, we wish to avoid the special danger that patents applied to a free program could make it effectively proprietary. To prevent this, the GPL assures that patents cannot be used to render the program non-free.
+
+The precise terms and conditions for copying, distribution and modification follow.
+
+TERMS AND CONDITIONS
+
+0. Definitions.
+
+�This License� refers to version 3 of the GNU General Public License.
```
Comment (thestr4ng3r) [inline LICENSES/GPL-3.0-or-later.txt:34]: These characters look strange.

Comment (karliss) [reply LICENSES/GPL-3.0-or-later.txt:34]: I Initially thought that it's github diff view is not playing nice with UTF8 but I just confirmed and the file really is using who knows what text encoding.

**Comment by xarkes** [comment]
Should we merge this?

**Comment by XVilka** [comment]
@xarkes is it incomplete. Will need cover some significant part of the source files to be ready. This one just adds exactly one file - `src/Main.cpp`.

**Pull request closed without merging**